scriptRunner/src/index.js

```diff
@@ -6,25 +6,25 @@ var app = express()
 require('dotenv').config()
 const config = require('../config')
 
-if (config.SENTRY_DSN) {
-    const Sentry = require('@sentry/node')
-    Sentry.init({
-        dsn: config.SENTRY_DSN,
-        release: require('../package.json').version
-    })
-}
+// if (config.SENTRY_DSN) {
+//     const Sentry = require('@sentry/node')
+//     Sentry.init({
+//         dsn: config.SENTRY_DSN,
+//         release: require('../package.json').version
+//     })
+// }
 
 // Constants
 const PORT = process.env.PORT || 9000;
 const HOST = process.env.NODE_ENV = "docker" ? '0.0.0.0' : 'localhost';
 
-const { getKeybaseImages } = require("./keybase")
-getKeybaseImages()
-setInterval(getKeybaseImages, 1000 * 60 * 5) // check once every 5 minutes for new validators (throttled by 24 after an update)
+// const { getKeybaseImages } = require("./keybase")
+// getKeybaseImages()
+// setInterval(getKeybaseImages, 1000 * 60 * 5) // check once every 5 minutes for new validators (throttled by 24 after an update)
 
-const { getTwitterImages } = require("./twitterImages")
-getTwitterImages()
-setInterval(getTwitterImages, 1000 * 60 * 5) // check once every 5 minutes for new validators (throttled by 24 after an update)
+// const { getTwitterImages } = require("./twitterImages")
+// getTwitterImages()
+// setInterval(getTwitterImages, 1000 * 60 * 5) // check once every 5 minutes for new validators (throttled by 24 after an update)
```

Comment:
this is the fix for rewards? Extracting keybase and twitter images was breaking them?

Reply:
no, I couldn't finish testing yesterday. will comment those lines in again